jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -163,6 +165,18 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    override fun setFeature(feature: EndpointDebugFeatures, enabled: Boolean) {
+        when (feature) {
+            EndpointDebugFeatures.PCAP_DUMP -> transceiver.setFeature(Features.TRANSCEIVER_PCAP_DUMP, enabled)
+        }
+    }
+
+    override fun isFeatureEnabled(feature: EndpointDebugFeatures): Boolean {
+        return when (feature) {
+            EndpointDebugFeatures.PCAP_DUMP -> transceiver.isFeatureEnabled(Features.TRANSCEIVER_PCAP_DUMP)
+        }
```

Comment:
You're no longer throwing an exception for an unsupported feature - is this deliberate?

Reply:
That was just for safety as a future fall through since there's only one enum value, but kotlin `when` errors when it isn't exhaustive so we don't need the fall through anymore.